GoogleForCreators/web-stories-wp
# Media: Improve list of supported file types for image pickers

## Context

Add supported mime types to file upload and media pickers to only support gif, png and jpg files. This means that an SVG can not be selected. 

## Summary



## Relevant Technical Choices

Add a new php method with a filter, so developers, can add mime types if they need to, but I am not sure if this should be in place. 


## To-do

- [ ] PHP unit tests

## User-facing changes



## Testing Instructions 

### QA






This PR can be tested by following these steps:

1. Dashboard settings.
2. Upload publisher logo. 
3. The file dialog should not allow you to select another other than a gif, jpg or png. 

In editor
1. Document tab
2. Select a Publisher logo or cover image. 
3. The file dialog should not allow you to select another other than a gif, jpg or png.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #5124
Fixes #6011
